Assets/HoloToolkit/Utilities/Scripts/Editor/CapabilitySettingsWindow.cs

```diff
@@ -8,7 +8,7 @@
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// Renders the UI and handles update logic for HoloToolkit/Configure/Apply Mixed Reality Capability Settings.
+    /// Renders the UI and handles update logic for Mixed Reality Toolkit/Configure/Apply Mixed Reality Capability Settings.
```

Comment:
Some of these changes are pending in another PR.

Reply:
Ahh okay. I'll revert this one.